### ISSUE TITLE
Https Protocol Not Supported

### ISSUE THREAD

**Issue by Eggy248** [Messere/listal-bot#1]
First of all, thank you for generating this tool. It comes in handy. It appears that recently listal.com went to the https protocol and the script no longer functions properly. I am not a programmer, but took a try at updating the code, but had no success. I imagine/hope this will be a quick tweak for you to make. Thanks again.

**Comment by Messere** [comment]
Hi Eggy, thanks for reporting.

While listal-bot itself does not care whether image links are secure or not, the problem is that node.js does not recognize certificate provider chosen by Listal (which is Alpha SSL). 

Fortunately, there's a workaround. 

1. Go to [AlphaSSL](https://www.alphassl.com/support/install-root-certificate.html) site and grab their certificate. This is the first one, labeled as `SHA-256 - Orders March 31, 2014 and After`. Copy/paste everything from the box (including `-----BEGIN CERTIFICATE-----` and `-----END CERTIFICATE-----`) to some text file, e.g. `alphassl.crt`.

2. Setup environment variable named `NODE_EXTRA_CA_CERTS` which points to the certificate file.  Now, how to do that depends on your OS.

For Linux / macOS :
```
export NODE_EXTRA_CA_CERTS="/path/to/your/alphassl.crt"
```
For Windows plain command line (`cmd`):
```
set NODE_EXTRA_CA_CERTS=C:\path\to\your\alphassl.crt
```

3. Try to run `listal-bot` again. It should no longer complain about certificate verification failure and do its job just fine. Unless you see any warnings at the very beginning, saying "ignoring extra certs" - double check the environment variable then. 

Note that those variables will be valid only in the same shell/command line. To set them permanently run 
(on Windows)
```
setx NODE_EXTRA_CA_CERTS C:\path\to\your\alphassl.crt
```
or add `export` statement to `~/.bashrc` file (on linux/macOS).

**Comment by Messere** [comment]
I've just published new version (0.5.4) which bundles missing certificate with the code. It should work without any workarounds.

Please upgrade (`npm update listal-bot -g`) and let me know if fix works for you.

cheers,
Messere

**Comment by Messere** [comment]
I assume it's working OK now (at least for me ;-)).
